include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.h

```diff
@@ -210,6 +210,12 @@ class MultigridPreconditionerBase : public PreconditionerBase<Number>
   unsigned int
   get_number_of_levels() const;
 
+  /*
```

Comment:
had to move `get_mapping()` from private to protected because one needs it within `include/exadg/incompressible_navier_stokes/preconditioners/multigrid_preconditioner_momentum.cpp` for the call of `MomentumOperator::initialize(..., mapping, ...)`, since there we now setup the variable viscosity models. The turbulence model requires the grid/element size.